sphinx/source/docs/reference/icons.rst

```diff
@@ -1,7 +0,0 @@
-.. _bokeh.icons:
-
-bokeh.icons
-===========
-
-.. automodule:: bokeh.icons
-  :members:
```

Comment:
If we leave a `bokeh/icons.py` with a warning, it can also have a module docstring that also discusses the removal, and points people at the example, in which case we should leave this sphinx doc for now. 

Suggest:

```
"""

.. warning::
    FontAwesome icons and the associated ``bokeh.icons`` modules were removed in 
    the Bokeh 0.12.4 release. Please see the :bokeh-tree:`examples/custom/font-awesome`
    directory for a demonstration of how to use these icons with a custom extension.

"""
import warnings
warnings.warn("FontAwesome icons and the associated bokeh.icons modules were removed in the Bokeh 0.12.4 release. Please see the 'examples/custom/font-awesome' directory for a demonstration of how to use these icons with a custom extension.")
del warnings
```